streams/src/test/java/org/apache/kafka/streams/integration/TableTableJoinIntegrationTest.java

```diff
@@ -55,8 +55,8 @@ public void prepareTopology() throws InterruptedException {
         appID = "table-table-join-integration-test";
 
         builder = new StreamsBuilder();
-        leftTable = builder.table(INPUT_TOPIC_LEFT);
-        rightTable = builder.table(INPUT_TOPIC_RIGHT);
+        leftTable = builder.table(INPUT_TOPIC_LEFT, Materialized.<Long, String, KeyValueStore<Bytes, byte[]>>as("left").withLoggingDisabled());
+        rightTable = builder.table(INPUT_TOPIC_RIGHT, Materialized.<Long, String, KeyValueStore<Bytes, byte[]>>as("right").withLoggingDisabled());
```

Comment:
Do we need to disable logging explicitly here?

Reply:
Yes. We want to not restore the stores in between tests (it is tricker to delete the changelogs since appId changes for each case, so I think this is easier).

Reply:
Ack.